BEIMA.Backend.Test/ReportService/ReportServiceTest.cs

```diff
@@ -0,0 +1,464 @@
+﻿using NUnit.Framework;
+using BEIMA.Backend.ReportService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BEIMA.Backend.MongoService;
+using Moq;
+using MongoDB.Bson;
+using System.IO.Compression;
+using System.IO;
+using MongoDB.Driver;
+
+namespace BEIMA.Backend.Test.ReportServices
+{
+    [TestFixture]
+    public class ReportServiceTest : UnitTestBase
```

Comment:
Let's add a test to make sure that headers and values are aligned properly in their respective columns, just to be safe.

Reply:
![image](https://user-images.githubusercontent.com/47906870/161161423-aa1dbd3c-2e51-4b8f-980c-eaa9a4ba39f4.png)

These statements are testing that. We are able to take advantage of the fact that Lists guarantee that the order in which items are added are how they will appear.  I'm using sub lists just to help separate out the difference properties (general, fields, location, modified).

Reply:
I suppose I was thinking more along the lines of checking something like, check that column 1 has this header and this value, then check that column 2 has this header and this value etc. So basically checking column by column instead of row by row. But this is more of just a sanity check I think, these tests should probably be good enough.